airbase/pom.xml

```diff
@@ -422,13 +429,6 @@
                 <version>${dep.joda.version}</version>
             </dependency>
 
-            <!-- misc stuff -->
-            <dependency>
-                <groupId>org.eclipse.jetty.toolchain</groupId>
-                <artifactId>jetty-jakarta-servlet-api</artifactId>
-                <version>5.0.2</version>
-            </dependency>
-
             <!-- servlet -->
```

Comment:
Remove this one too?

Reply:
There are more javax.* dependencies to remove. I've added the removal in a separate commit.